### ISSUE TITLE
Communication with API's

### ISSUE THREAD

**Issue by ovflowd** [sant0ro/espreso#21]
Something really nice it's Espreso communicating with Chocolatey by an External API.

Not a Habbo API, but a _MUS_ Like API, Espreso will manage the entire system by managing those API's. And will authenticate by Tokens.

Is this a good idea? What your opinion @Emeto ?

**Comment by T0niiiiii** [comment]
So you mean Arcturus rcon?

**Comment by ovflowd** [comment]
Nope. Read more carefully. It's chocolatey communicating to espresso.
